src/main/java/teammates/ui/pagedata/InstructorFeedbackResponseCommentAjaxPageData.java

```diff
@@ -40,7 +40,12 @@ public FeedbackResponseCommentRow getComment() {
                                                showCommentToString, showGiverNameToString,
                                                getResponseVisibilities(), instructorEmailNameTable, sessionTimeZone);
         frc.enableEditDelete();
-
+        String whoCanSeeComment;
```

Comment:
Declare this in line 46?

Keep the scope as small as possible. Since we only need it in the `if` part, declare and initialize there itself